bsp/stm32/stm32f091-st-nucleo/README.md

```diff
@@ -29,6 +31,8 @@ The mainly-used resources of this board are shown as follows:
 - Common-used interfaces: USB, Arduino interface, etc.
 - Debug interface: standard SWD For more details about this board, please refer to [ST official](https://www.st.com/en/evaluation-tools/stm32-nucleo-boards.html?querycriteria=productId=LN1847).
```

Comment:
"S“ of Standard should take the upper case and add a period at the end of the sentence.

Put an interval before "For more details"